### PR TITLE
Changing file permissions for securityconfig files

### PR DESCRIPTION
**Pull request by debjanibnrj**
*Issue #, if available:* 

*Description of changes:* Updated securityconfig yml file permissions to disable files being world-readable and executable by default.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/main/assemblies/plugin.xml [hunk 1]
```diff
@@ -27,10 +27,17 @@
             <outputDirectory>${file.separator}</outputDirectory>
             <includes>
                 <include>tools/**</include>
-                <include>securityconfig/**</include>
             </includes>
             <fileMode>0755</fileMode>
```
Comment (vrozov) [inline src/main/assemblies/plugin.xml:31]: Should it be 750?

Comment (debjanibnrj) [reply src/main/assemblies/plugin.xml]: updated to 750.

**Comment by debjanibnrj** [comment]
Testing steps

* Setup opendistro v1.6
* Install latest security plugin version and exec install_demo_configuration.sh
* Delete security index
* Restart ES process
* Call REST Apis to see if internal_users, roles, rolesmapping and action_groups are loaded.

curl -k -XGET --cacert root-ca.pem --key kirk-key.pem --cert kirk.pem -u admin:admin https://localhost:9200/_opendistro/_security/api/internal_users

**Comment by sujithvm** [comment]
@debjanibnrj 

> Testing steps
> 
> * Setup opendistro v1.6
> * Install latest security plugin version and exec install_demo_configuration.sh
> * Delete security index
> * Restart ES process
> * Call REST Apis to see if internal_users, roles, rolesmapping and action_groups are loaded.
> 
> curl -k -XGET --cacert root-ca.pem --key kirk-key.pem --cert kirk.pem -u admin:admin https://localhost:9200/_opendistro/_security/api/internal_users

This does not run the security admin tool right? This uploads from yaml if opendistro index does not exist. 

Wanted to check paths like 
- https://github.com/opendistro-for-elasticsearch/security/blob/master/src/main/java/com/amazon/opendistroforelasticsearch/security/tools/OpenDistroSecurityAdmin.java#L909 
- https://github.com/opendistro-for-elasticsearch/security/blob/master/src/main/java/com/amazon/opendistroforelasticsearch/security/tools/Migrater.java#L89

Just want to cross-check that file writes are not affected as security admin tool is run as console app (with main method) and not within ES process

**Comment by debjanibnrj** [comment]
@sujithvm: Also tested using the securityadmin tool directly and updated roles mapping from /securityconfig/roles.yml successfully. 


```
./securityadmin.sh -f ../securityconfig/roles.yml -t roles -cert ../../../config/kirk.pem -cacert ../../../config/root-ca.pem -key ../../../config/kirk-key.pem 
Open Distro Security Admin v7
Will connect to localhost:9300 ... done
Connected as CN=kirk,OU=client,O=client,L=test,C=de
Elasticsearch Version: 7.6.1
Open Distro Security Version: 1.6.0.0
Contacting elasticsearch cluster 'elasticsearch' and wait for YELLOW clusterstate ...
Clustername: elasticsearch
Clusterstate: YELLOW
Number of nodes: 1
Number of data nodes: 1
.opendistro_security index already exists, so we do not need to create one.
Populate config from /Users/bdebjani/workspace/opendistro-es/es/es7.6/opendistroforelasticsearch-1.6.0/plugins/opendistro_security/tools
Force type: roles
Will update '_doc/roles' with ../securityconfig/roles.yml 
   SUCC: Configuration for 'roles' created or updated
Done with success
```

```
curl -k -XGET --cacert root-ca.pem --key kirk-key.pem --cert kirk.pem -u admin:admin https://localhost:9200/_opendistro/_security/api/roles | python -m json.tool | grep new_ -A10

    "new_role": {
        "cluster_permissions": [],
        "hidden": false,
        "index_permissions": [
            {
                "allowed_actions": [
                    "crud"
                ],
                "fls": [],
                "index_patterns": [
                    ".opendistro-alerting-config",
```

**Pull request merged**